provenance.md

```diff
@@ -0,0 +1,24 @@
+# Provenance
+[See issues](https://github.com/OCR-D/spec/issues/11)
+All provenance have to be stored in files. 
+There's provenance at the page level, but there's also provenance at the document level.
+Because these files can become very large, they are also stored as references in the fileGrp section.
+
+## Format
+Not defined yet
+
+### Input/Output
+Input should be the input files referenced via FileIDs. Optionally the parameter file.
+Output should be the generated files referenced via FileIDs.
+
+### Ingest/Access Workspace to/from OCR-D Repositorium
+The linking of inputs and outputs should be done via the FileID as long as the files are not archived. 
+If the files are in repository, the FileIDs should be replaced by the unique URL.
+When exporting a workspace, the references should be reset to the FileID again.
+During the ingest, the entire provenance must be stored in a graph to make the provenance searchable.
+
+## METS
+### File Group
+File groups holding provenance have to start with prefix "PROV-"
```

Comment:
See above, I'm not convinced whether provenance and logging data should be part of METS at all.

I think it would be sufficient to record `mets:agent` for the workflow that touched a METS with a URL to a workflow engine run which then allows further introspection.

Reply:
Yes I agree, it should be tracked but should it be part of the payload? Or wouldn't it make more sense to have

* in the METS only a reference to the URL representing the workflow execution(s) that modified this file with OCR-D tools. Using a mets:agent for example.

* Make that URL dereferenceable for provenance and logging metadata

* Bundle the data in the container (bag) but not the METS file itself?

Reply:
I always struggle with the term metadata, because PAGE XML is also called metadata in other use cases. 
We may also define a tag directory metadata which contains a provenance and log directory holding all collected metadata. But it is difficult to refer to it from the METS file because it is a parent directory.
For provenance metadata we may reference the input/output files via their ID inside the METS file. The log metadata must be referenced using a job ID that should be part of the provenance metadata.